client/converters/FlightConfirmation.jsx

```diff
@@ -0,0 +1,38 @@
+export const flightConfirmationConverter = ({ flight }) => {
```

Comment:
Converters can help us (especially in Redux) convert data from state to props without having to clog up our `mapStateToProps` functions. Here, I'm using it in a basic React way, and maybe could be better. But I wanted to start the general pattern.